static/styles/left_sidebar.css

```diff
@@ -221,8 +221,8 @@ li.show-more-topics {
     }
 }
 
+#login-link,
 #subscribe-to-more-streams {
-    text-decoration: none;
```

Comment:
Why this change? It seems like it should be a separate commit, since it's a styling change, not a content one.